.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,52 @@
+name: run-tests
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: 3.6
```

Comment:
Do you mean to just run this on python-3.6? or also 3.7, 3.8 like on Jenkins?

Reply:
These are done via Tox environments, 3.6 is just to run the `python -m tox` command which will create the 3.6, 3.7 and 3.8 environments.